### PR TITLE
Add banme skill

### PR DESCRIPTION
**Pull request by getjump**
/banme 🔫

### CODE REVIEW

File: bot/skills/__init__.py [hunk 1]
```diff
@@ -54,10 +56,12 @@ def _make_skill(add_handlers: Callable, name: str, hint: str) -> Dict:
     _make_skill(add_uwu, "😾 uwu", " don't uwu!"),
     _make_skill(add_mute, "🤭 mute", " mute user for N minutes"),
     _make_skill(add_roll, "🔫 roll", " life is so cruel... isn't it?"),
+    _make_skill(add_banme, "🔫 banme", " Commit Sudoku"),
```
Comment (C-Pro) [inline bot/skills/__init__.py]: Didn't knew that sudoku has so dire consequences!

File: bot/skills/mute.py [hunk 2]
```diff
@@ -13,29 +13,44 @@
...
-    dp.add_handler(CommandHandler("mute", mute, filters=admin_filter), handlers_group)
+    dp.add_handler(CommandHandler("mute", mute, filters=admin_filter), 
+                   handlers_group)
 
 
 def _get_minutes(args: List[str]):
     # cmd should be a reply for going to mute user message like "/mute 90"
     if len(args) < 1:
-        raise Exception("mute cmd should be a reply for going to mute user message like '/mute 90', "
+        raise Exception("mute cmd should be a reply for going to mute user"
```
Comment (egregors) [inline bot/skills/mute.py]: redundant formatting

File: bot/skills/mute.py [hunk 3]
```diff
@@ -13,29 +13,44 @@
...
+                        " message like '/mute 90', "
                         "where '90' is duration of the mute")
     return int(args[0])
 
 
-def mute(update: Update, context: CallbackContext):
+def mute_user_for_time(update: Update, context: CallbackContext,
```
Comment (egregors) [inline bot/skills/mute.py]: redundant formatting

File: bot/skills/mute.py [hunk 4]
```diff
@@ -13,29 +13,44 @@
...
+                       user: User, mute_minutes: int):
     try:
-        mute_minutes = _get_minutes(context.args)
         until = datetime.now() + timedelta(minutes=mute_minutes)
-        user: User = update.message.reply_to_message.from_user
-        logger.info(f"user: {user.full_name}[{user.id}] will be muted for {mute_minutes} min")
 
-        update.message.reply_text(f"Таймаут для {user.full_name} на {mute_minutes} минут")
-        context.bot.restrict_chat_member(update.effective_chat.id, user.id, until,
+        logger.info(f'''user: {user.full_name}[{user.id}]
```
Comment (egregors) [inline bot/skills/mute.py]: redundant formatting

**Comment by egregors** [comment]
rel #68

**Comment by egregors** [comment]
Clean and nice PR! Thanks @getjump 🔥 🔥 🔥 🔥 🔥

**Pull request merged**